### PR TITLE
LCR-1883: Implementation of LCR-1883

### PR DESCRIPTION
**Pull request by timj**

### CODE REVIEW

File: LSE-61.tex [hunk 1]
```diff
@@ -327,14 +327,20 @@ \subsubsection{Guider Calibration Data Acquisition}
...
+
+For visits resulting in fewer than \textbf{nAlertVisitPeak}, LSST shall be capable of supporting the distribution of at least \textbf{OTR1} per cent of alerts via the LSST alert distribution system within time \textbf{OTT1} from the conclusion of the camera's readout of the raw exposures used to generate each alert.
 
-\textbf{Discussion:} Because of the processing flow of SSObject orbit determination, meeting the base 24-hours-after-data-acquisition requirement would be far more challenging than for the other L1 Data Products, but the system throughput has to be good enough such that a back log can not build up.
+Solar System Object orbits will, on average, be calculated before the following night's observing finishes and the results \textit{shall} be made available within time \textbf{L1PublicT} of those calculations being completed.
+
+\textbf{Discussion:}
+    Because of the processing flow of SSObject orbit determination, meeting the base \textbf{L1PublicT}-hours-after-data-acquisition requirement would be far more challenging than for the other L1 Data Products, but the system throughput has to be good enough such that a back log can not build up.
```
Comment (gpdf) [inline LSE-61.tex]: *L1PublicT* is a quantity with units.  "hours" should not appear after it.

File: LSE-61.tex [hunk 2]
```diff
@@ -359,6 +365,28 @@ \subsubsection{Nightly Data Accessible Within 24 hrs}
...
+98
+&
+\unitname{%
+percent
+}
+&
+\paramname{%
+OTR1
+} \\\hline
+The instantaneous peak number of alerts per standard visit.
```
Comment (gpdf) [inline LSE-61.tex]: "Instantaneous" is not an ideal word here.  OTR1 is not a rate but a quantity.

Comment (MelissaGraham) [reply LSE-61.tex]: Hmm, I do think of it as a rate: number per visit (number of alerts per 35 seconds). The term "instantaneous" is useful to differentiate nAlertVisitAvg from nAlertVisitPeak. Perhaps "The instantaneous peak rate of alerts per standard visit" is a better definition of nAlertVisitPeak?

File: LSE-61.tex (hunk 1 above)
Comment (timj) [reply LSE-61.tex]: I've updated the model to remove the hours.

**Comment by timj** [comment]
This PR ended up including changes from multiple change requests because trunk was updated with a lot of them in short order: LCR-1883, LCR-2004, LCR-2008, LCR-1203, LCR-1915, LCR-1554

**Pull request merged**